lib/templates/gitignore.erb

```diff
@@ -0,0 +1 @@
+"/<%= slug %>.md"
```

Comment:
won't this result in the markdown file not making it into the curriculum repo? I'm fairly certain gitignores recurse through the file system.